<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>751</width>
     <height>791</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="segmentTab">
    <attribute name="title">
     <string>Segment</string>
    </attribute>
    <widget class="QLabel" name="markersImagePixLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>360</width>
       <height>360</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>No Markers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="markersImageTitle">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Markers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>550</y>
       <width>591</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="optionsLayout">
      <item>
       <layout class="QVBoxLayout" name="segmentMethodLayout">
        <item>
         <widget class="QLabel" name="segmentLabel">
          <property name="text">
           <string>Segment Method</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="segmentMethodComboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Otsu (Global) Binarization</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Local Threshold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pixel Classifier</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ensemble</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Detectron2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Segment Anything</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="edgeMethodLayout">
        <item>
         <widget class="QLabel" name="edgeLabel">
          <property name="text">
           <string>Edge Detection Method</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="edgeMethodMethodComboBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Local Thresholding</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bright-Dark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Testing</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="segmentImageTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Segment&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="segmentImagePixLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>360</width>
       <height>360</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>No Threshold</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="runSegmentationPushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>710</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveSegmentationButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>710</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save .h5</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="classifyTab">
    <attribute name="title">
     <string>Classify</string>
    </attribute>
    <widget class="QLabel" name="classifyRegionPixLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string>Load Image First...</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backClassifyButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardClassifyButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>280</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelingGuideTitleLabel">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>540</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Labeling Guide&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelingGuideBodyLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>580</y>
       <width>571</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting for input image </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="labelingGuideLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>670</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveClassifyButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>720</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Labeled csv...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="inputImagePixLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>230</y>
     <width>361</width>
     <height>360</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drag-and-Drop or&lt;/p&gt;&lt;p&gt;Select Below&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inputImageTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectInputPushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>240</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image or Folder...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputLeftPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputRightPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="topBoundInputLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>690</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftBoundInputLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="bottomBoundInputLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>790</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>860</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightBoundInputLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>740</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2560</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleInputLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>740</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>9.37</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="topBoundInputLabel">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>670</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Top Bound</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bottomBoundInputLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>770</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom Bound</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rightBoundInputLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>720</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Bound</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="leftBoundInputLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Bound</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scaleInputLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>720</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>px to μm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
